runtime/queries/bitbake/highlights.scm

```diff
@@ -99,150 +111,167 @@
 ; Reset highlighting in f-string interpolations
 (interpolation) @none
 
-;; Identifier naming conventions
+; Identifier naming conventions
 ((python_identifier) @type
- (#lua-match? @type "^[A-Z].*[a-z]"))
-([(identifier) (python_identifier)] @constant
- (#lua-match? @constant "^[A-Z][A-Z_0-9]*$"))
+  (#lua-match? @type "^[A-Z].*[a-z]"))
+
+([
+  (identifier)
+  (python_identifier)
+] @constant
+  (#lua-match? @constant "^[A-Z][A-Z_0-9]*$"))
 
 ((python_identifier) @constant.builtin
- (#lua-match? @constant.builtin "^__[a-zA-Z0-9_]*__$"))
+  (#lua-match? @constant.builtin "^__[a-zA-Z0-9_]*__$"))
 
 ((python_identifier) @constant.builtin
- (#any-of? @constant.builtin
-           ;; https://docs.python.org/3/library/constants.html
-           "NotImplemented"
-           "Ellipsis"
-           "quit"
-           "exit"
-           "copyright"
-           "credits"
-           "license"))
+  (#any-of? @constant.builtin ; https://docs.python.org/3/library/constants.html
+   "NotImplemented" "Ellipsis" "quit" "exit" "copyright" "credits" "license"))
```

Comment:
```suggestion
  ; format-ignore
  (#any-of? @constant.builtin
   ; https://docs.python.org/3/library/constants.html
   "NotImplemented" "Ellipsis" "quit" "exit" "copyright" "credits" "license"))
```

Reply:
These changes should also be applied to python and starlark.

Reply:
Done.